src/header/common/authorization.rs

```diff
@@ -1,6 +1,7 @@
 use std::fmt;
 use std::str::{FromStr, from_utf8};
 use std::ops::{Deref, DerefMut};
+use std::marker::Reflect;
```

Comment:
Reflect shouldn't be needed in this file as far as I can tell. What error comes up if you remove these bounds?